src/App.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components/macro'
+import useFormData from './hooks/useFormData'
+import SaveGameForm from './components/SaveGameForm'
+import GameCard from './components/GameCard'
+
+
+function App() {
+
+  const { gameProfile, setGameProfile, savedGameProfiles, setSavedGameProfiles } = useFormData()
+ 
+  return (
+    <AppWrapper>
+      <SaveGameForm gameProfile={gameProfile} setGameProfile={setGameProfile} savedGameProfiles={savedGameProfiles} setSavedGameProfiles={setSavedGameProfiles} />
+      <GameCardListWrapper>
+        {savedGameProfiles.map(({location, date, players, winner, shots, _id}) => 
+          <GameCard key={_id} location={location} date={date} players={players} winner={winner} shots={shots}/>
+        )}
+      </GameCardListWrapper>
+
+    </AppWrapper>
+  );
+}
+
+export default App;
+
+
+const AppWrapper = styled.div`
+  display: grid;
+  grid-gap: 20px;
+`
+const GameCardListWrapper = styled.div`
```

Comment:
I prefer to style the child elements within the out element style (in this case `AppWrapper`) using sass to reduce that amout of strange long wrapper-styled-whatever names

Reply:
``GameCardListWrapper`` is a wrapper for all ``GameCards``, that will be rendered. That is why I called it "list". But it is indeed confusing :D I'll rename it to ``GameCardsWrapper``. Hope this is better to understand.